NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -160,7 +161,15 @@ class NeuroID
                     configTimeout = 10,
                 )
 
-            configService = NIDConfigService(dispatcher, logger, this, httpService, validationService)
+            configService =
+                NIDConfigService(
+                    dispatcher,
+                    logger,
+                    this,
+                    httpService,
+                    validationService,
+                    configRetrievalCallback = { configSetupCompletion() },
+                )
             samplingService = NIDSamplingService(logger, randomGenerator, configService)
```

Comment:
I see where you made changes to sampling service elsewhere, but any reason to still leave this sampling service invoked here?

Reply:
Because we will be reimplementing the sample logic in a future release and we didn't want to rip out our work we left this and made the value a constant.